core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -271,7 +327,27 @@ case class MultiClassificationMetrics
   Recall: Double,
   F1: Double,
   Error: Double,
-  ThresholdMetrics: MulticlassThresholdMetrics
+  ThresholdMetrics: MulticlassThresholdMetrics,
+  TopKMetrics: MultiClassificationMetricsTopK
+) extends EvaluationMetrics
+
+/**
+ * Metrics for topK MultiClassification
+ *
+ * Each metric contains a list of metrics corresponding to each of the topK most occurring labels.  If the predicted
+ * label is outside of the topK most occurring labels, it is treated as incorrect.
+ *
+ * @param Precision
+ * @param Recall
+ * @param F1
+ * @param Error
+ */
+case class MultiClassificationMetricsTopK
```

Comment:
We also should keep track of the topKs here so that we don't need to get it from the evaluator params